internal/flickr/handlers/util.go

```diff
@@ -0,0 +1,32 @@
+package handlers
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/anz-bank/sysl-go/common"
+)
+
+//  return a string ptr
```

Comment:
Nit: Comments should be complete sentences, starting with the name of the thing they're describing (https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences). They should also be *de*scriptive ("X does Y"), not *pre*scriptive ("Do Y").

```suggestion
// newString returns a string pointer.
```

and below

Reply:
ok